tests/data/meta/allowed_values_2.yaml

```
@@ -0,0 +1,4 @@
+- Meta cat with boolean values:
+  allowed_values: [True, False]

```

In keeping consistency with the other CodeLists `allowed_values` should be indented like so:

```suggestion
    allowed_values: [True, False]
```
you'd need to slightly change the way you read the yaml file but I'd say it's worth keeping consistency.